ext/js/dom/text-source-range.js

```diff
@@ -187,9 +189,14 @@ export class TextSourceRange {
     /**
      * Selects the text source in the document.
      */
+    
     select() {
```

Comment:
Does this event trigger anytime the popup pops up? Suppose I held down the shift key and passed through a sentence and each time a popup pops for every word along the sentence at ~100ms/word, would that be a bunch of "fake" selects?